labinstructions.md

```diff
@@ -150,6 +151,7 @@ There is no reason to use the same instrumentation key for the JavaScript and se
     4. Replace `instrumentation_key_for_aspnet_app` in the inserted snippet to the actual instrumentation key
 
 3. Open [http://localhost:24002](http://localhost:24002) in the browser and click couple links to generate some telemetry. It takes some time for telemetry to show up on server. You'll see telemetry from JavaScript in the next exercise.
+4. With Chrome browser, install auto-refresh extension from here: https://chrome.google.com/webstore/detail/auto-refresh/ifooldnmmcmlbdennkpdnlnbgbmfalko?hl=en-US and configure it to refresh http://localhost:24002/Home/Details?stock=msft every 3 seconds. This way you generate telemetry.
```

Comment:
I don't think it's a good idea to install extension. First, I was using Edge and we do not have instruction to use Chrome. Second, long links may not work well as copy/paste is not reliable in lab environment.

I found that there were permissions issues, not unload of application. So constant telemetry flow shouldn't be a problem.